.github/workflows/action.yml

```diff
@@ -0,0 +1,36 @@
+name: 'Build Matlab bindings'
+description: 'Installs preCICE and builds the preCICE Matlab bindings in the directory. Ends in `$GITHUB_WORKSPACE/matlab-bindings`'
+
+inputs:
+  matlab-release:
+    required: true
+    default: 'latest'
+    description: 'The release of Matlab to use.'
+  precice-version:
+    required: true
+    default: '2.4.0'
```

Comment:
Shouldn't this be 2.5.0 ?

Comment, not critical: On the develop branch of the Matlab bindings ideally you would always test against the latest version of preCICE (in order to fail early). But I don't know whether this is supported upstream. You might have to build from source for this to work.